src/Command/DrupalUpdate/DrupalPackageManager.php

```diff
@@ -0,0 +1,314 @@
+        }
+        if ( trim($row) == "version" ) {
+          $current_version = trim($project_value);
+        }
+        if ( trim($row) == "package" ) {
+          $package_alternative_name = strtolower(trim($project_value));
+        }
+      }
+    }
+
+    if ($current_version == 'VERSION') {
+      $current_version = $this->determineCorePackageVersion($drupal_project_cwd_path);
+    }
+    if ($package_type == '') {
+      $package_type = ($package_alternative_name == 'core') ? 'drupal' : '';
+    }
+    if ( ($this->isCoreUpdated === FALSE) || ($package_type !== 'drupal') ) {
+      if (trim($package_type) === '') {
+        return;
+      }
+      $package_available_releases_data=$drupal_client->getSecurityRelease(trim($package_type), $current_version);
+      if (is_array($package_available_releases_data) & !empty($package_available_releases_data)) {
+        $package_name = key($package_available_releases_data);
+        $this->packageData[$package_name] = $package_available_releases_data[$package_name];
+      }
+    }
+    if ($package_type == 'drupal') {
+      $this->isCoreUpdated = TRUE;
+    }
+  }
+
+  /**
+   * @param $version_detail
+   */
+  public function printPackageDetail($version_detail): void {
+    $table = new Table($this->output);
+    $git_commit_message_detail = [];
+    array_shift($version_detail);
+    $array_keys = array_column($version_detail, 'package');
+    array_multisort($array_keys, SORT_ASC, $version_detail);
+
+    foreach ($version_detail as $versions) {
+      $package = $versions['package'];
+      $git_commit_message = [];
+      $git_commit_message[] = $package;
+      $git_commit_message[] = $versions['package_type'];
+      $git_commit_message[] = $versions['current_version'] ?? '';
+      $git_commit_message[] = $versions['latest_version'] ?? '';
+      $git_commit_message[] = $versions['update_notes'] ?? '';
+      $git_commit_message_detail[] = $git_commit_message;
+    }
+    $table->setHeaders([
+          'Package Name',
+          'Package Type',
+          'Current Version',
+          'Latest Version',
+          'Update Type',
+      ])->setRows($git_commit_message_detail);
+    $table->render();
+  }
+
+  /**
+   * Get Packages detail information.
+   * Package name, package type, package current version etc.
+   * @param string $drupal_project_cwd_path
+   * @return array
+   * @throws AcquiaCliException
+   */
+  public function getPackagesMetaData(string $drupal_project_cwd_path): array {
+    $this->io->note('Checking available updates.');
+    $this->infoPackageFilesPath = $this->fileSystemUtility->getInfoFilesList($drupal_project_cwd_path);
+    if (count($this->infoPackageFilesPath) == 0) {
+      throw new AcquiaCliException("Not valid Drupal 7 project.");
+    }
+    $this->io->note('Preparing packages.');
+    $this->getPackageDetailInfo($this->infoPackageFilesPath, $drupal_project_cwd_path);
+    return $this->availablePackageUpdatesList($drupal_project_cwd_path);
```

Comment:
availablePackageUpdatesList - if the function returns something, its name should start with get. If it does something, its name should start with a verb.